admin/fleetcommander/admin.py

```diff
@@ -627,20 +627,37 @@ def websocket_stop(self):
             c.websocket_stop(self.current_session['websockify_pid'])
             del(self.current_session['websockify_pid'])
 
+
 if __name__ == '__main__':
 
-    # Python import
     from argparse import ArgumentParser
-
-    # Fleet commander imports
     from utils import parse_config
 
+    # Fleet commander constants
+    import constants
+
     parser = ArgumentParser(description='Admin interface server')
     parser.add_argument(
+        'host', action='store', metavar='HOST',
+        help='Listening host for admin application')
+    parser.add_argument(
         '--configuration', action='store', metavar='CONFIGFILE', default=None,
         help='Provide a configuration file path for the web service')
+    parser.add_argument(
+        '--port', action='store', metavar='PORT', type=int, default=constants.FC_ADMIN_STANDALONE_PORT,
+        help='Listening port for admin application')
+    parser.add_argument(
+        '--appdir', action='store', metavar='APPDIR', default=constants.FC_ADMIN_APP_DIR,
+        help='Directory for application data')
+    parser.add_argument(
+        '--statedir', action='store', metavar='STATEDIR', default=None,
+        help='Directory for data storage')
```

Comment:
I would use --profiledir and 'Profile storage directory', 'data storage' and statedir are too ambiguous